src/Menu.js

```diff
@@ -0,0 +1,52 @@
+import './Menu.css';
+import {useState} from "react";
+import AlertPage from "./AlertPage";
+
+function Menu(props) {
+
+    const [menuDisplay, setMenuDisplay] = useState(false);
+
+    function toggleMenuDisplay() {
+        setMenuDisplay(!menuDisplay)
+    }
+
+    function hideCompleted() {
+        props.toggleCompletedDisplay(false)
```

Comment:
completedDisplay and menuDisplay seem as though they share the same value. If so, no need for separate state variables.